index.rst

```diff
@@ -10,20 +10,849 @@
+However, moving the Antu servers to the summit enables this processing to occur in the event of an outage to the base.
+Speeding up this process, if required, would necessitate processing the data at the USDF, which is planning real-time support for commissioning (see `RTN-021 <https://rtn-021.lsst.io>`_).
+Although this does not explicitly include donut analysis, the cluster is fully capable of doing so and would not be running other real-time analysis at that time.
+A trigger to process the AOS data would be required, how this would get accomplished is under investigation.
+Discussions are currently ongoing with Richard Dubois to better define the needed support and required timeline(s).
+
+Therefore, FAFF ultimately recommends moving the Antu servers to the summit; the technical details are currently being captured in `ITTN-061 <https://ittn-061.lsst.io>`_.
+This will add functionality in the case of an outage and decreases the workload of cluster management and maintenance by co-locating the hardware and removing one set of services.
+If the compute load is insufficient to perform all Rapid Analysis tasks, then the project can either augment the number of machines, or reduce the number of detectors that are processed in the pipeline.
+In discussions with both the AOS and Science Verification teams, using ~50% of the detector has not been met with any resistance.
+If full-focal plane wavefront sensing requires more compute, we recommend moving that processing to the USDF and developing an automatic trigger mechanism.
+In the case where the link to USDF is lost, it will be required to accept the additional overhead associated with performing the calculation on fewer machines (the Antu servers) [#]_, which is the originally baselined plan.
+
+
+.. [#] A single full focal plane analysis currently takes ~3 min with 2 cores per chip. Note that Rapid Analysis does not need to be run on these images, thus saving compute time, but it is important to make sure the processes are setup such that they do not compete.
+
+
+.. _Deliverable 6:
+
+Deliverable 6: Camera Visualization Tool Expansion Support
+-----------------------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+  6. Develop a plan and scope estimate to expand the Camera Visualization Tool to support the full commissioning effort.
+
+     This includes identifying libraries/packages/dependencies that require improvements (e.g. Seadragon) and fully scoping what is required to implement the tool with DM tooling such as the Butler.
+     The scope estimate may propose the use of in-kind contribution(s) to this effort if and where applicable.
+
+We have developed a plan to address the visualization requirements developed as part of first FAFF report and further refined based on
+discussion during FAFF2. The plans include the following major categories:
```

Comment:
"discussion during FAFF2." should be in the previous line and the new line should start in "The"